packages/compass-e2e-tests/helpers/commands/open-aggregation-side-panel.ts

```diff
@@ -0,0 +1,12 @@
+import type { CompassBrowser } from '../compass-browser';
+import * as Selectors from '../selectors';
+
+export async function openAggregationSidePanel(
```

Comment:
should we make this toggle and then we unify open and close methods?

Reply:
I had them separate because it is convenient to trigger an individual state explicitly. If what you're suggesting is `toggleSidePanel` with an extra arg (`state: 'open' | 'close'`) then that works too.

Reply:
I just did that. Could you please check again?